qe_metrics/cli.py

```diff
@@ -9,10 +8,34 @@
 from qe_metrics.libs.jira import Jira
 from qe_metrics.utils.issue_utils import create_update_issues
 from qe_metrics.utils.product_utils import products_from_file
+from pyhelper_utils.runners import function_runner_with_pdb
+
 
 LOGGER = get_logger(name="main-qe-metrics")
 
 
+@orm.db_session
```

Comment:
@calebevans Do I need this on both functions?

Reply:
I think you only need it on this one (qe_metrics) @myakove

Reply:
If you setup a local config from the readme and run, it will throw an error if you need it.

Reply:
I will check it

Reply:
Seems to work without any